projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -36,6 +48,20 @@ const Admin = () => {
 		);
 	}
 
+	if ( isShareLimitEnabled && ! hasPaidPlan && showPricingPage ) {
+		return (
+			<AdminPage moduleName={ __( 'Jetpack Social 1.0', 'jetpack-social' ) } header={ <Logo /> }>
```

Comment:
Do you think we could prevent repeating this? The contents of the one below could be wrapped in a fragment perhaps.

Should we take this opportunity to make the version number dynamic? We could define the `moduleName` as a const earlier on.

Reply:
Done in [5181979](https://github.com/Automattic/jetpack/pull/26213/commits/5181979cf81db3adf903f4828aa7d67098a2fd7c).